.github/stale.yml

```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 28
```

Comment:
Commit message says 45 and here's 28, please match them :)

Reply:
Done! Thanks :)